cpp/IceStorm/clock/Subscriber.cpp

```diff
@@ -168,7 +168,7 @@ main(int argc, char* argv[])
     // Add a servant to the object adapter. If --id is used the identity comes from the command line, otherwise a UUID
     // is used. id is not directly altered since it is used below to detect whether subscribeAndGetPublisher can throw
     // AlreadySubscribed.
-    Ice::Identity subId{.name = id.empty() ? Ice::generateUUID() : id};
+    Ice::Identity subId{id.empty() ? Ice::generateUUID() : id, ""};
```

Comment:
Is category `""` required? I thought Bernard recently fixed this.

Reply:
We can't use designed initializer syntax  as it's c++20 and windows would complain. Bernard did recently fix it but it  was really for GCC. Windows won't let you do it.

Reply:
The issue here is we're building in C++17 mode on all platforms and VC++ doesn't support designated initializers in C++17 mode.